src/integrationtests/resources/com/aws/greengrass/integrationtests/logmanager/smallSpaceManagementPeriodicIntervalConfig.yaml

```diff
@@ -10,7 +10,7 @@ services:
             minimumLogLevel: 'INFO'
             diskSpaceLimit: '105'
             diskSpaceLimitUnit: 'KB'
-            deleteLogFileAfterCloudUpload: 'true'
+            deleteLogFileAfterCloudUpload: 'false'
```

Comment:
This is only used for diskSpaceManagement Test, where files should not be processed.